chromium_src/services/network/DEPS

```diff
@@ -1,5 +1,6 @@
 include_rules = [
   "+../../../../services/network",
+  "+../../../../../../services/network",
```

Comment:
is this one correct?

Reply:
It's needed for the include in https://github.com/brave/brave-core/blob/cr92/chromium_src/services/network/public/cpp/url_util.cc, but on a second thought it would probably be clearer to have a DEPS file in chromium_src/services/network/public/cpp instead.

Reply:
Fixed in https://github.com/brave/brave-core/commit/8b6bff3d77ff6d7d1464fb63a092cb0110508e3a